python/lsst/meas/astrom/matchOptimisticB/matchOptimisticBContinued.py

```diff
@@ -286,6 +287,12 @@ def matchObjectsToSources(self, refCat, sourceCat, wcs, refFluxField, maxMatchDi
             self.log.info("filterStars purged %d reference stars, leaving %d stars" %
                           (preNumObj - numRefObj, numRefObj))
 
+        if toleranceStruct is None:
+            toleranceStruct = pipeBase.Struct(
+                maxMatchDist=None,
+            )
```

Comment:
Would this be better to create as its own 1-2 story ticket after this ticket is committed? I could see us creating it and then use it as a slightly bigger restructure that stores not just the tolerances, but the matches themselves and methods for computing statistics on the matches.